src/ms-review.tex

```diff
@@ -0,0 +1,1786 @@
+\end{figure}
+
+% itemize because it helps me write, could also just be paragraphs...
+\subsubsection{WCS Maps}
+The FITS WCS pixelization supports a different number of projections to
+represent celestial spherical coordinates in a regular rectangular grid.
+\gammapy provides full support to data structures using this pixelization
+scheme. For details see ~\cite{Calabretta2002}. This pixelisation
+is typically used for smaller regions of interests, such as pointed
+observations.
+
+
+\subsubsection{HEALPix Maps}
+This pixelization scheme ~\citep{Calabretta2002} provides a
+subdivision of a sphere in which each pixel covers the same surface area as
+every other pixel. As a consequence, however, pixel shapes are no longer
+rectangular, or regular.
+This pixelisation is typically used for all-sky data, such as data
+from the \hawc or \fermi observatory. \gammapy natively supports
+the multiscale definiton of the HEALPix pixelisation and thus
+allows for easy up and downsampling of the data. In addition to
+the all-sky map, \gammapy also supports a local HEALPix
+pixelisation where the size of the map is constrained to a given
+radius.
+For local nighbourhood operations, such as convolution \gammapy relies
+on projecting the HEALPix data to a local tangential WCS grid.
+
+\subsubsection{Region Maps}
+In this case, instead of a fine spatial grid
+dividing a rectangular sky region, the spatial dimension is reduced to a single
+bin with an arbitrary shape, describing a region in the sky with that same
+shape. Typically they are is used together with a non-spatial dimension, for
+example an energy axis, to represent how a quantity varies in that dimension
+inside the corresponding region. The region is represented on the local
+tangential projection see \cite{Astropy regions}.
+
+Additionally, the \code{MapAxis} class provides a uniform API for axes representing
+bins on any physical quantity, such as energy or angular offset. The special
+case of time is covered by the dedicated \code{TimeMapAxis}, which allows time bins to
+be non-contiguous, as it is often the case with observation time-stamps. The
+generic class \code{LabelMapAxis} allows the creation of axes for non-numeric
+entries.
+
+\subsection{gammapy.datasets}
+\label{ssec:gammapy-datasets}
+%
+\begin{figure}
+	\import{code-examples/generated/}{gp_datasets}
+	\caption{
+        Using \code{gammapy.datasets} to read existing reduced binned datasets.
+        After the different datasets are read from disk they are collected into a
+        common \code{Datasets} container. All dataset types have an associated
+        name attribute to allow access by name later in the code. The
+        environment variable \code{\$GAMMAPY\_DATA} is automtically resolved
+        by \gammapy.
+    }
+	\label{fig*:minted:gp_datasets}
+\end{figure}
+%
+The end product of the data reduction process described in Section
+\ref{ssec:gammapy-makers} are a set of binned counts, background and IRF maps,
```

Comment:
the backgound is an IRF (see the corresponding sub-section).... May I propose "binned counts and IRF maps (such as the background map),